conanfile.py

```diff
@@ -43,8 +43,8 @@ def _make_spec(self):
 
     def configure(self):
         # It's not a C project, but libcxx is hardcoded in the project
-        del self.settings.compiler.libcxx
-        del self.settings.compiler.cppstd
+        # del self.settings.compiler.libcxx
+        # del self.settings.compiler.cppstd
```

Comment:
```suggestion
        # del self.settings.compiler.cppstd
        pass
```
this shouldfix the IndentationError